src/commands/e2e/E2E.ts

```diff
@@ -224,6 +230,23 @@ export class E2E implements ICommand {
         return true;
     }
 
+    private filterPluginsByHasSpecifiedTests(): void {
+        this.pluginsToBeTested = this.pluginsToBeTested
+            .filter((plugin) => {
+                        const pathToSpecFiles = path.join(this.workingDir, plugin, 'assets', 'e2e', 'specs');
+                        if (fs.existsSync(pathToSpecFiles)) {
+                            if (glob.sync(path.join(pathToSpecFiles, '**', '*.spec.ts')).length > 0) {
+                                Global.isVerbose() && console.warn(`Plugin ${plugin} has specified .spec.ts test`);
+                                return true;
+                            }
+                        } else {
```

Comment:
The `else` still has to go, otherwise the case  were the directory exists but no test-files are there will not return false - or (and?) you need to append the two if-statements.